cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -465,17 +471,19 @@ def create_route_spec(connection, routes, vpc_id):
 def ensure_route_table_present(connection, module):
 
     lookup = module.params.get('lookup')
-    propagating_vgw_ids = module.params.get('propagating_vgw_ids', [])
+    propagating_vgw_ids = module.params.get('propagating_vgw_ids')
```

Comment:
This is good; the default is supposed to be `None` anyway as per the module spec.